src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -9,10 +9,8 @@
 using System.Collections.Immutable;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Linq;
```

Comment:
It doesn't look like this file has any meaningful changes. I recommend reverting it to its original state.